controllers/cre/controller.go

```diff
@@ -109,10 +110,11 @@ func (r *CustomRuntimeEnvironmentReconciler) reconcilePipelineRun(ctx context.Co
 	pipeline := build_types[cre.Spec.BuildType]
 	pipelineRun := &pipelinev1beta1.PipelineRun{
 		ObjectMeta: metav1.ObjectMeta{
-			Name:      fmt.Sprintf("cre-%s-%s", cre.GetName(), pipeline),
-			Namespace: cre.Namespace,
+			GenerateName: fmt.Sprintf("cre-%s-%s-", cre.GetName(), pipeline),
```

Comment:
We should check if a PipelineRun already exists for this CRE

Reply:
Would it be okay to to add the generation directly to the name? Like:
```
Name: fmt.Sprintf("cre-%s-%s-%s", cre.GetName(), cre.GetGeneration(), pipeline)
```

Reply:
Let's conclude discussions in #136

Reply:
@KPostOffice generation in the pipeline name can't hurt, but we would also need a label with the generation if we choose that scheme, so we can query with label the K8S api.